scripts/add-sibling.js

```diff
@@ -66,10 +68,3 @@ var indexPath = path.join(basePath, 'packages', 'all-packages', 'src', 'index.ts
 var index = fs.readFileSync(indexPath, 'utf8');
 index = index + 'import "' + package.name + '";\n';
 fs.writeFileSync(indexPath, index);
-
-// Add the extension to jupyterlab/package.json
-var jupyterlabPackagePath = path.join(basePath, 'jupyterlab', 'package.json');
```

Comment:
Thanks for the heads up.  `package.json` and `package.app.json` are mean for dev mode and app mode, respectively.  The latter is copy of `package.json` at the time of publishing the Python package.